lib/metrics.py

```diff
@@ -124,51 +125,53 @@ def __compare_ssim(planes):
   a, b = planes
   if a is None or b is None: # handle Y800 case
     return 1.0
-  return skimage.measure.compare_ssim(a, b, win_size = 3)
+  return skimage.metrics.structural_similarity(a, b, win_size = 3)
```

Comment:
Why the function name change?  Did it change in newer version of skimage?  If so, please support both skimage versions during import.  That is, if user has old version installed, then it should still work.  You can assign the appropriate function to a global function pointer variable depending on the imported version, then just use the global variable function pointer here.

Reply:
It looks like several of skimage.measure functions are deprecated since 0.16 (https://scikit-image.org/docs/dev/api_changes.html#version-0-16) .  We should do something like:

```python
try:
  # try skimage >= 0.16, first
  from skimage.metrics import structural_similarity as skimage_ssim
except:
  from skimage.measure import compare_ssim as skimage_ssim
```

...then in `__compare_ssim` we just call `skimage_ssim`.

The same needs to be done for mse, nrmse and psnr metrics, too (https://scikit-image.org/docs/dev/release_notes_and_installation.html#deprecations).

Additionally, this change has nothing to do with py2to3 porting.  Please submit as a separate pull-request.

Reply:
will do the change

Reply:
Another PR for this: #145, please help to review it.